cumulus/parachains/runtimes/assets/asset-hub-westend/src/lib.rs

```diff
@@ -1117,6 +1212,75 @@ impl pallet_migrations::Config for Runtime {
 	type WeightInfo = weights::pallet_migrations::WeightInfo<Runtime>;
 }
 
+parameter_types! {
+	pub MaximumSchedulerWeight: frame_support::weights::Weight = Perbill::from_percent(80) *
```

Comment:
80% seems to be a lot

Reply:
I think 80% makes sense when assuming 5MiB proof size, to allow for 4MiB runtimes. For 10MiB it is probably too much.